gmso/core/topology.py

```diff
@@ -369,11 +370,25 @@ def add_connection(self, connection, update_types=True):
         update_types : bool, default=True
             If True also add any Potential object associated with connection to the
             topology.
+
+        Returns
+        _______
+        Connection
+            The Connection object or equivalent Connection object that
+            is in the topology
         """
+        # Check if an equivalent connection is in the topology
+        equivalent_members = connection.get_equivalent_members()
+        if equivalent_members in self._unique_connections:
+            warnings.warn('An equivalent connection already exists.')
```

Comment:
I think it would make sense to give one more sentence about the process happening under the hood.

```python3
warnings.warn('An equivalent connection already exists. Providing the equivalent Connection.')
```
Or something to that effect.